app/src/main/java/com/github/multimatum_team/multimatum/service/ProcrastinationDetectorService.kt

```diff
@@ -93,14 +192,48 @@ class ProcrastinationDetectorService : Service(), SensorEventListener {
          * The sensor used by this service to detect movements
          */
         const val REF_SENSOR = Sensor.TYPE_GRAVITY
-      
-        private const val MIN_PERIOD_BETWEEN_NOTIF_NANOSEC = 2_000_000_000L
+
+        private const val MIN_PERIOD_BETWEEN_NOTIF_NANOSEC = 2_000_000_000L  // 2 seconds
 
```

Comment:
ahy this is noted as a change ?

Reply:
Because of the comment that I added at the end of the line